src/test/java/org/mifos/integrationtest/cucumber/stepdef/ValidateChannelDef.java

```diff
@@ -122,53 +122,49 @@ public void iCreateATransactionChannelRequestDTO() {
 
     @When("I call the post transfer API having unsupported header with expected status of {int}")
     public void iCallThePostTransferAPIHavingUnsupportedHeaderWithExpectedStatusOf(int expectedStatus) {
-            RequestSpecification requestSpec = Utils.getDefaultSpec();
-            scenarioScopeState.tenant = tenant;
-            scenarioScopeState.response = RestAssured.given(requestSpec).header("Content-Type", "application/json")
-                    .header(tenantHeader, scenarioScopeState.tenant).header("invalid-header", "test")
-                    .baseUri(channelConnectorConfig.channelConnectorContactPoint)
-                    .body(scenarioScopeState.createTransactionChannelRequestBody).expect()
-                    .spec(new ResponseSpecBuilder().expectStatusCode(expectedStatus).build()).when()
-                    .post(channelConnectorConfig.transferEndpoint).andReturn().asString();
-
-            logger.info("Post Transfer Response: {}", scenarioScopeState.response);
+        RequestSpecification requestSpec = Utils.getDefaultSpec();
+        scenarioScopeState.tenant = tenant;
+        scenarioScopeState.response = RestAssured.given(requestSpec).header("Content-Type", "application/json")
```

Comment:
use constants

Reply:
This files were changed because of spotless those are not new feature introduced by me